clouddrift/adapters/subsurface.py

```diff
@@ -92,7 +92,9 @@ def to_xarray(
             "index_exp": (["traj"], metadata["indexExp"]),
             "float_type": (["traj"], metadata["fltType"]),
             "id": (["traj"], metadata["indexFlt"]),
-            "date": (["obs"], data["dtnum"]),
+            "rowsize": (["traj"], rowsize),
+            "datenum": (["obs"], data["dtnum"]),
+            "ids": (["obs"], np.repeat(metadata["indexFlt"], rowsize)),
```

Comment:
I suggest not including `ids` in the output dataset as it's not needed by `cd.ragged.subset` anymore and is readily computable.

Reply:
🫡 it's gone.